tests/unit/source_code/test_graph.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
+
+import pytest
+from databricks.labs.ucx.source_code.graph import Dependency, DependencyGraph, DependencyResolver, SourceContainer, \
+    WrappingLoader, DependencyProblem
+from databricks.labs.ucx.source_code.files import FileLoader, LocalFileResolver
+
+from tests.unit import _load_sources
+
+
+def test_dependency_path():
+    """The dependency path should return the path."""
+    path = Path("test")
+    dependency = Dependency(FileLoader(), path)
+    assert dependency.path == path
+
+
+@pytest.fixture()
+def file_dependency() -> Dependency:
+    yield Dependency(FileLoader(), Path("test"))
+
+
+def test_wrapping_loader_load_dependency(mock_path_lookup, file_dependency):
+    """Should return the source container"""
+    sources = _load_sources(SourceContainer, "root4.py.txt")
```

Comment:
@ericvergnaud : Why does load sources return strings? I would expect it to return source containers

Reply:
Don't use _load_sources, use MockPathLookup